src/munge.ts

```diff
@@ -40,6 +41,15 @@ export function disableRemb(sdp: Sdp) {
   disableRtcpFbValue(sdp, 'goog-remb');
 }
 
+/**
+ * Disable REMB from all media blocks in the given SDP.
```

Comment:
Typo here (should say TWCC instead of REMB).

Reply:
udpated.